src/components/SelectedBeer.js

```diff
@@ -0,0 +1,40 @@
+import React, {useState, useEffect} from 'react'
+import axios from 'axios'
+import Header from './Header' 
+
+function SelectedBeer(props) {
+     const [selectedBeer, setBeerState] = useState({})
+     const beerInfo = props.match.params.beerId
+
+     useEffect(() => {
+        axios.get(`https://ih-beers-api2.herokuapp.com/beers/${beerInfo}`)
+        .then(selectedBeer => {
+            setBeerState(selectedBeer.data)
+        })
+        .catch((error)=> "Error while fetching the beer from API")
+       })
+     
+    
+    return (
+        <div>
+            <h1>Selected Beer</h1>
+            <Header/>
```

Comment:
You need to give a value to the prop `location` otherwise the image won't render.